### DISCUSSION TITLE
timeline for this process

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#66] [Q&A] [answered]
Does anyone know what the timeline and deadlines are for this process?

The date of Jan. 1, 2022 for files being up on websites is clearly in the rule, but what is the goal for when there will be a final specification for the data formats.

Also, is the intention that the format as specified here is what must be complied with or is it a recommendation for how to present the data?  Or is it part of the process of defining what is in the guidance.

The rule states that the "The machine-readable files described in this paragraph (b) must be available in
a form and manner as specified in guidance issued by the Department of the Treasury, the Department of Labor, and the Department of Health and Human Services."

Is the "Code" section of this GitHub the guidance?

**Comment by ghost** [comment]
Great question! I posted a similar question "Machine Readable Files Schema Release Freeze Date? #45" 6 days ago and am awaiting a reply. 

From the organizations I have spoken with, everyone is basing their file schema and associated guidance off this site for their design and development work. That is why there is concern based on the methods being developed to generate a In-Network Negotiated Rates file of any changes to the schema and associated guidance since it is the more complex file as compared to Allowed-Amounts file based on the rule.

**Reply by nmayle** [reply]
I think a schedule would also be useful for any individuals or organizations that are thinking of contributing to the process or want to make sure the specification is complete and reasonable.

**Reply by ryan-thomas-peterson** [reply]
Agree at the need for a a schedule to be laid out so folks can reasonably develop a more refined plan of attack.

**Reply by tdfow** [reply]
With so many open/unanswered discussions,  is there any plans for CMS to provide any extensions to the the 1/1/2022 date?  Or how are others planning to address those areas that are still unresolved?

**Comment by shaselton-usds** [comment, marked as answer]
Everyone on this thread probably has been paying attention, but incase that isn't the case, this was asked in Discussion: https://github.com/CMSgov/price-transparency-guide/discussions/45. An answer has been provided there.
